pkg/crc/machine/machine.go

```diff
@@ -424,6 +432,20 @@ func (client *client) Start(startConfig StartConfig) (StartResult, error) {
 	}, err
 }
 
+func getSSHPort(vsockNetwork bool) int {
```

Comment:
not a huge fan of passing in a boolean to control the behaviour, but also not really of a static variable...